int8 clean
int8 pump
int8 left
int8 right
int8 mop
int8 park
int8 forward
int8 backward
int8 wash
int8 speed_enable
int8 driverless
int8 driverless_enabled
float32 velocity
float32 angle
float32 angle_velocity
float32 charge

 
